docs/pages/blog/mui-core-v5.md

```diff
@@ -0,0 +1,546 @@
+   The styled API forces to constantly jump between the usage of the styled components and where they are defined.
+   Could we move the style descriptions right where we need them?
+2. **Naming things** is hard.
+   Have you ever found yourself struggling to find a good name for a styled component?
+   Could we remove the need to create and name yet another component?
+3. **Enforcing consistency** in UIs is hard.
+   This is especially true when more than one person is building the application, as there has to be some coordination amongst members of the team regarding the choice of design tokens and how they are used, what parts of the theme structure should be used with what CSS properties, and so on.
+
+In v5, we have pushed the solution one step further with the `sx` prop.
+The prop is now available on **all** the components thanks to the migration away from JSS.
+It exposes as a superset of the CSS API, the normal CSS properties, shorthands, and media query helpers.
+
+For instance, you can add two vertical margin units with:
+
+```jsx
+// add margin: 16px 0px;
+<Slider sx={{ my: 2 }} />
+```
+
+<p class="blog-description"><a href="https://codesandbox.io/s/nostalgic-williams-zmo5r?file=/src/App.js">Codesandbox</a></p>
+
+Developers seem to already [love it](https://twitter.com/AnsonLowZF/status/1397034690771443715).
+You can find a [side-by-side comparison](/system/basics/#why-use-the-system) of `styled` vs. `sx` in the documentation to determine when you want to use the prop.
+Find where your cursor is. Some developers use it for everything, others with parsimony.
+
+The four components categorized as CSS utilities: Box, Grid, Typography, and Stack also expose a subset of the `sx` prop as flat props, for instance:
+
+```jsx
+<Typography color="grey.600">
+```
+
+<p class="blog-description"><a href="https://codesandbox.io/s/keen-worker-zo2r3?file=/src/App.tsx">Codesandbox</a></p>
+
+See the [API tradeoff](/system/basics/#api-tradeoff) section of the documentation for why not all the components accept these flat props.
+
+### Dynamic props
+
+React is about composition. Developers can import one component, extend it, and re-export the wrapper.
+It's how developers have extended our core components up until v4. However:
+
+1. Each time you create a new component, it's another import option for your team.
+   Now, you have to enforce that the right component is imported.
+2. Adding a new `color="success"` prop to a Button component requires non-trivial CSS customizations.
+   How do you ensure that all the styles (hover, focus, focus-visible) are consistent with the other built-in colors?
+3. It adds a boilerplate.
+
+It's why v5 comes with the capability to extend the built-in behavior of the components; right from the theme.
+It was one of the most upvoted GitHub issues: [#13875](https://github.com/mui-org/material-ui/issues/13875).
+In practice, the change makes the MUI Core components extendable placeholders.
```

Comment:
> MUI Core components

Off topic: we seriously need to reconsider the naming of `@mui/core`...

Reply:
Yeah, I kept the Core terminology because of https://material-ui.com/components/about-the-lab/#about-the-lab

> The main difference between the lab and the core is how the components are versioned.